net/ftp-proxy/src/opnsense/mvc/app/controllers/OPNsense/FtpProxy/Api/ServiceController.php

```diff
@@ -62,7 +59,7 @@ public function statusAction($uuid)
     public function startAction($uuid)
     {
         $result = array("result" => "failed", "function" => "start");
-        if ($this->request->isPost()) {
+        if (isset($this->request) && $this->request->isPost()) {
             $this->sessionClose();
         }
```

Comment:
```suggestion
```

Reply:
Sheesh you were faster than me. I just got yelled at by git in the command line xD